lib/encoder.js

```diff
@@ -622,12 +622,17 @@ function JPEGEncoder(quality) {
       y = 0;
     var r, g, b;
     var start, p, col, row, pos;
-    var chunkOffset = -chunkHeight;
-    var imageData;
-    while (y < height) {
+    var chunkOffset = 0;
+    var img = await getRows(chunkOffset, 512);
```

Comment:
What's wrong with having 512 be a parameter?

Reply:
uh - nothing that seems good.

Reply:
done - sorry I moved some names around and lost it.